bin/qbatch

```diff
@@ -29,11 +29,10 @@ SHELL = os.environ.get("QBATCH_SHELL", "/bin/sh")
 OPTIONS = [os.environ.get("QBATCH_OPTIONS")] if os.environ.get(
     "QBATCH_OPTIONS") else []
 
-if os.getenv('SGE_ROOT'):
-    if os.environ.get("QBATCH_SYSTEM", "sge") or os.environ.get("QBATCH_SYSTEM", "slurm"):
-        SYSTEM = os.environ.get("QBATCH_SYSTEM")
-    else:
-        sys.exit("specify either sge or slurm for QBATCH_SYSTEM")
+if subprocess.call(['which', 'sbatch'], stdout=open(os.devnull))==0:
```

Comment:
Hrm, well maybe that definition should move... 

This is partially an issue with qbatch currently being a "script" style program that we should really rewrite to be a "class" style program.